core/src/tangram.cpp

```diff
@@ -29,6 +29,7 @@
 #include <bitset>
 #include <mutex>
 #include <queue>
+#include <array>
```

Comment:
std::array is used in tangram.cpp - could be that some header changes removed the indirect include